README.md

```diff
@@ -367,11 +367,11 @@ interface XRWebGLBinding {
 
   readonly attribute double nativeProjectionScaleFactor;
 
-  XRProjectionLayer createProjectionLayer(XRTextureType textureType, XRProjectionLayerInit init);
-  XRQuadLayer createQuadLayer(XRTextureType textureType, XRLayerInit init);
-  XRCylinderLayer createCylinderLayer(XRTextureType textureType, XRLayerInit init);
-  XREquirectLayer createEquirectLayer(XRTextureType textureType, XRLayerInit init);
-  XRCubeLayer createCubeLayer(XRLayerInit init);
+  XRProjectionLayer createProjectionLayer(XRTextureType textureType, optional XRProjectionLayerInit init = {});
+  XRQuadLayer createQuadLayer(XRTextureType textureType, optional XRLayerInit init = {});
```

Comment:
Only XRProjectionLayerInit has all default parameters so the init can be omitted. XRLayerInit needs a pixel width and height.

Reply:
Oops, good point. Fixed.